transformers/events/vat_flux/converter_test.go

```diff
@@ -19,23 +19,41 @@ package vat_flux_test
 import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
 	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
 	"github.com/makerdao/vdb-mcd-transformers/transformers/events/vat_flux"
-	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
 	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
 	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
 )
 
 var _ = Describe("VatFlux converter", func() {
-	var converter = vat_flux.VatFluxConverter{}
-	It("Converts logs to models", func() {
-		models, err := converter.ToModels(constants.VatABI(), []core.HeaderSyncLog{test_data.VatFluxHeaderSyncLog})
+	var (
+		converter vat_flux.Converter
+		db        *postgres.DB
+	)
+
+	BeforeEach(func() {
+		converter = vat_flux.Converter{}
+		db = test_config.NewTestDB(test_config.NewTestNode())
```

Comment:
Maybe initialize the db in the `var` declaration so it only happens once per suite